tests/e2e/report-submission.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { test, expect, chromium } from "@playwright/test"
+import login from "./utils/login"
+
+test.describe("Authenticated Dispatch App", () => {
+  test.beforeEach(async ({ page }) => {
+    await login(page)
+  })
+  test("Should allow me to report an incident", async ({ page }) => {
+    /* The ability to report an incident is one of the most critical
+      user stories in the Dispatch application. */
+
+    const project = "Test"
+    const type = "Customer Data"
+    const priority = "Low"
+
+    await page.getByRole("link", { name: "Report Incident" }).click()
+
+    // Title
+    await page.getByLabel("Title").click()
+    await page.getByLabel("Title").fill("Incident Test Created by Playwright")
+
+    // Description
+    await page.getByLabel("Description").click()
+    await page.getByLabel("Description").fill("test")
+
+    // Project
+    await page.getByRole("combobox").filter({ hasText: "Project" }).locator("i").click()
+    await page.getByText(project, { exact: true }).click()
+
+    // Type
+    await page.getByRole("button", { name: "Type" }).click()
+    await page.getByText(type, { exact: true }).click()
+
+    // Priority
+    await page.getByRole("button", { name: "Priority" }).click()
+    await page.getByText(priority).click()
+
+    // Open the dropdown selection for the Tags field
+    await page.getByRole("combobox").filter({ hasText: "Tags" }).locator("i").click()
+
+    // Soft check that the 'Load More' selector is available upon opening the Tags dropdown
+    await expect
+      .soft(page.getByText("Load More"), "The 'Load More' selector should be visibile.")
```

Comment:
```suggestion
      .soft(page.getByText("Load More"), "The 'Load More' selector should be visible when there are more than 5 options in the Tags combobox.")
```

Also, can we check if there's more than 5 options in a smart way first to make this a conditional check?